sormas-e2e-tests/src/main/java/org/sormas/e2etests/enums/cases/epidemiologicalData/TypeOfActivityActivity.java

```diff
@@ -0,0 +1,28 @@
+package org.sormas.e2etests.enums.cases.epidemiologicalData;
+
+public enum TypeOfActivityActivity {
```

Comment:
Can we find another name that will not duplicate Activity?

Reply:
if is the same, we can use the existent one